### DISCUSSION TITLE
db size error in etcd version 3.5.8

### DISCUSSION THREAD

**Discussion by ssengar** [etcd-io/etcd#16366] [Q&A] [answered]
### What would you like to be added?

after switching to etcd ver 3.5.8 ...started getting mvcc: database space exceeded error...in 3 weeks(my quota-backend-bytes is 6GB)
previously was running 3.4.13 from last 2 year and this error never same. want to know what extra this 3.5.8 version is storing.

### Why is this needed?

after switching to etcd ver 3.5.8 ...started getting _**mvcc: database space exceeded error**_...in 3 weeks(my quota-backend-bytes is 6GB)
previously was running 3.4.13 from last 2 year and this error never same. want to know what extra this 3.5.8 version is storing.

**Comment by ahrtr** [comment, marked as answer]
Please read https://etcd.io/blog/2023/how_to_debug_large_db_size_issue/

**Comment by ssengar** [comment]
Hi @ahrtr ,

does etcd store any other type of internal data in db file which is not pushed by etcd clients (ex: putting a key value pair).
Just want to understand what all type of data is stored in etcd by default.

**Reply by ahrtr** [reply]
> does etcd store any other type of internal data in db file which is not pushed by etcd clients (ex: putting a key value pair).

NO. All key-value data in the key space are put by clients. Please figure out what data consume most of the storage space per the above guide.
